frontend/locales/nl.json

```
@@ -5,5 +5,46 @@
   "Faculty of Humanities": "Faculteit der Geesteswetenschappen",
   "Not logged in": "Niet ingelogd",
   "MyResearch": "MyResearch",
-  "Welcome!": "Welkom!"
-}
+  "Welcome!": "Welkom!",
+  "Home": "Home",
+  "What is MyResearch?": "Wat is MyResearch?",
+  "Roadmap": "Wegwijzer",
+  "Processing register": "Verwerkingsregister",
+  "Ethical commission": "Ethische commissie",
+  "Recent Activity": "Recente Activiteit",
+  "Frequent questions": "Veelgestelde vragen",
+  "Other questions": "Overige vragen",
+  "My Research is a website that helps researchers with submitting a request to start a research. On this website you can register your research at the processing register. But first you should fill in the roadmap": "My research is een website dat onderzoekers helpt met het aanvragen van onderzoek. Op deze website kan uw onderzoek worden geregistreerd bij het verwerkingsregister. Maar eerst is het de bedoeling dat een wegwijzer wordt ingevuld.",
+  "The roadmap is a questionnaire that helps researchers to find out what needs to happen to submit a request to start a research. Do I need to register my research at processing register at all. Perhaps that isn't needed? Questions that the roadmap helps answering": "De wegwijzer is een vragenlijst dat onderzoekers helpt uit te vogelen wat er moet gebeuren om een onderzoek aan te vragen. Moet ik voor mijn onderzoek nou wel of niet een mijn gegevens registeren in het verwerkingsregister? Mogelijk hoeft het niet eens. Iets wat de wegwijzer samen met u uitzoekt. ",
+  "Start a new roadmap": "Start een nieuwe wegwijzer",
```

We only have one Roadmap, right?

```suggestion
  "Start the Roadmap": "Start de Wegwijzer",
```

---

We are saving one roadmap per research no? The questions stay the same but a researcher can have multiple roadmap results.